smoke_testing/test_scan.py

```
@@ -62,16 +58,25 @@ def scan_all_format_pdf(kubescape_exec: str):
+        except Exception as e:
+            return f"{e}"
 
 
 def run(kubescape_exec: str):
     print("Testing E2E on yaml files")
-
-    # TODO - fix support
-    # print("Testing scan all yaml files")
-    # msg = scan_all(kubescape_exec=kubescape_exec)
-    # smoke_utils.assertion(msg)
+    smoke_utils.assertion(msg)
```

Ruff (0.16.1)

[error] Undefined name `msg`

(F821)